src/Stryker.CLI/Stryker.CLI.UnitTest/StrykerCLITests.cs

```diff
@@ -377,301 +124,224 @@ public void StrykerCLI_OnMutationScoreAboveThresholdBreak_ShouldReturnExitCode0(
-        [InlineData("-url https://www.example.com/")]
-        public void ShouldOverwriteDefaultDashboardUrlWhenPassed(string argName)
+        [InlineData("--mutate")]
+        [InlineData("-m")]
+        public void ShouldPassFilePatternSetToStryker_WithMutateArgs(string argName)
         {
-            StrykerOptions options = null;
-            var runResults = new StrykerRunResult(new StrykerOptions(), 0.3);
-
-            var mock = new Mock<IStrykerRunner>(MockBehavior.Strict);
-            mock.Setup(x => x.RunMutationTest(It.IsAny<StrykerOptions>(), It.IsAny<IEnumerable<LogMessage>>()))
-                .Callback<StrykerOptions, IEnumerable<LogMessage>>((c, m) => options = c)
-                .Returns(runResults)
-                .Verifiable();
-
-            var target = new StrykerCLI(mock.Object);
+            var firstFileToExclude = "**/*Service.cs";
+            var secondFileToExclude = "!**/MySpecialService.cs";
+            var thirdFileToExclude = "**/MyOtherService.cs{1..10}{32..45}";
 
-            target.Run(new string[] { argName, "--reporters", "['dashboard']", "--dashboard-project", "test", "--dashboard-api-key", "test" });
+            target.Run(new[] { argName, firstFileToExclude, argName, secondFileToExclude, argName, thirdFileToExclude });
 
-            mock.VerifyAll();
+            strykerRunnerMock.VerifyAll();
 
-            options.DashboardUrl.ShouldBe("https://www.example.com/");
+            var filePatterns = inputs.MutateInput.SuppliedInput.ToArray();
+            filePatterns.Length.ShouldBe(3);
+            filePatterns.ShouldContain(firstFileToExclude);
+            filePatterns.ShouldContain(secondFileToExclude);
+            filePatterns.ShouldContain(thirdFileToExclude);
         }
 
-        [Fact]
-        public void ShouldKeepDefaultDashboardUrlWhenArgumentNotProvided()
+        [Theory]
+        [InlineData("--since")]
+        [InlineData("-since")]
+        public void ShouldEnableDiffFeatureWhenPassed(string argName)
         {
-            StrykerOptions options = null;
-            var runResults = new StrykerRunResult(new StrykerOptions(), 0.3);
-
-            var mock = new Mock<IStrykerRunner>(MockBehavior.Strict);
-            mock.Setup(x => x.RunMutationTest(It.IsAny<StrykerOptions>(), It.IsAny<IEnumerable<LogMessage>>()))
-                .Callback<StrykerOptions, IEnumerable<LogMessage>>((c, m) => options = c)
-                .Returns(runResults)
-                .Verifiable();
-
-            var target = new StrykerCLI(mock.Object);
-
-            target.Run(new string[] { "--reporters", "['dashboard']", "--dashboard-project", "test", "--dashboard-api-key", "test" });
+            target.Run(new string[] { argName });
 
-            mock.VerifyAll();
+            strykerRunnerMock.VerifyAll();
 
-            options.DashboardUrl.ShouldBe("https://dashboard.stryker-mutator.io");
+            inputs.SinceInput.SuppliedInput.Value.ShouldBeTrue();
         }
 
```

Comment:
Anything to verify the default branch to compare to? And, on that note, have you considered changing `master` to `main` there? GitHub has made this the new default branch name I think.

Reply:
I think `master` is still generally most used. But indeed more and more teams are switching to `main`. Maybe `main` would be more future proof since it might become most used. But good idea, we could check if a `master` branch exists. If not look for a `main` branch.

Reply:
Many more projects will use master as the default for the foreseeable future imo. At some point we can switch but right now I think using master as default will have a larger impact.

Reply:
Well I guess master will be prevalent for a couple of year although I like the idea with the fallback master->main.

Reply:
Fallback with master -> main is difficult because we currently have no way to see if a user did not specify any option (so the default was used) or a user explicitly set it to master. It is a good idea but it would require a redesign.

Reply:
We can check for existing branches in the SinceTargetInput? We set the default there only when no user input was supplied. I'll create a issue for it

Reply:
We need to be very sure we can do that safely, but yes that's possible.